src/main/scala/com/fasterxml/jackson/module/scala/ScalaObjectMapper.scala

```diff
@@ -0,0 +1,372 @@
+// scalastyle:off
+
+/**
+ * This trait was copied verbatim from: https://github.com/FasterXML/jackson-module-scala/blob/2.10/src/main/scala/com/fasterxml/jackson/module/scala/ScalaObjectMapper.scala.
+ * This trait establishes forward compatibility with jackson-module-scala 2.10+.
+ *
+ * The below license was copied from: https://github.com/FasterXML/jackson-module-scala/blob/2.10/src/main/resources/META-INF/LICENSE
+ */
+
+/**
+ * This copy of Jackson JSON processor Scala module is licensed under the
+ * Apache (Software) License, version 2.0 ("the License").
+ * See the License for details about distribution rights, and the
+ * specific rights regarding derivate works.
+ *
+ * You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package com.fasterxml.jackson.module.scala
```

Comment:
Can we put this under `com.fasterxml.jackson.module.scala.hyperspace`? Otherwise, we may hit into singer mismatch error.

Reply:
or `com.fasterxml.jackson.hyperspace.module.scala`

Reply:
Changed.